cves/kernel/CVE-2014-0101.yml

```
@@ -56,6 +51,7 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  Vulnerability CVE-2014-0101 allows attackers to exploit the Linux kernel's implementation of the SCTP protocol (a networking protocol), The issue lies spesifically in the 'sctp_sf_do_5_1D_ce' in the kernel's code, which doesn't properly check certain values before performing a particular operation. An attacker can exploit the vulnerability remotely by sending specially crafted SCTP packets to a system that is vulnerable. The packets can lead to memory corruption and denial of service (NULL pointer dereference and system crash).
```

1.
```suggestion
  Vulnerability CVE-2014-0101 allows attackers to exploit the Linux kernel's implementation of the SCTP protocol (a networking protocol), The issue lies specifically in the 'sctp_sf_do_5_1D_ce' function in the kernel's code, which doesn't properly check certain values before performing a particular operation. An attacker can exploit the vulnerability remotely by sending specially crafted SCTP packets to a system that is vulnerable. The packets can lead to memory corruption and denial of service (NULL pointer dereference and system crash).
```
2. Can you clarify what operation it is performing when you say "before performing a particular operation"?
3. Can you clarify about "sending specially crafted SCTP packets to a system that is vulnerable"? What is the system that it's sending it to and why is that system vulnerable?

---

Thank you for the feedback and after rereading I agree that the description was lacking. Ended up rewriting it